testData/solutions/win32.json5

```diff
@@ -3002,6 +3002,613 @@
         ]
     },
 
+    "com.microsoft.windows.internationalization": {
+        "name": "Windows Cursors",
```

Comment:
That name is already taken

Reply:
Ups, changing it. (also that could explain some weird behavior I experienced 😅 )